requirements.txt

```diff
@@ -1,14 +1,48 @@
+astroid==3.0.3
 certifi==2023.11.17
+charset-normalizer==3.3.2
 click==8.1.7
+dateparser==1.2.0
+DateTime==5.4
+decorator==5.1.1
+dill==0.3.7
 elastic-transport==8.11.0
 elasticsearch==8.11.1
 elasticsearch7==7.13.0
+expandvars==0.6.5
 fmatch==0.0.4
-numpy==1.26.3
-pandas==2.1.4
+gevent==23.9.1
+greenlet==3.0.3
+hunter @ git+https://github.com/datastax-labs/hunter.git@8ff166979d000780ad548e49f006ef2a15d54123
+idna==3.6
+isort==5.13.2
+mccabe==0.7.0
+more-itertools==8.14.0
```

Comment:
Thank you for the suggestion Paige. But I believe having the inter dependencies added to our requirements will help avoid any future conflicts.
https://stackoverflow.com/questions/48526360/should-we-put-all-required-and-their-dependent-packages-on-requirement-txt-or-on

Reply:
I dont think that helps us in this current situation. I think the issue is if we have hard requirements (ex `more-itertools==8.14.0`) in our requirements, but we don't actually use this package in our code and hunter then updates their requirements to `more-itertools==8.15.0`, we will hit versioning issues until we install our requirements. 
I think dependency management will help but I think our requirements should only install the packages we outright import in our python scripts to avoid issues

Reply:
I understand your concern, however, in a case where we update hunter to a newer version, it'll also update `more-itertools`. At the moment the responsibility of dependency management lies on us i.e. we need to update requirements.txt whenever we install a new package or a new version of a package. I understand we need to update our packages from the new requirements whenever there is a version bump of what we use, but I believe this will make sure there are no dependency conflicts in the future

Reply:
I was able to de-clutter our requirements.txt based on @paigerube14 's recommendation 

```
hunter @ git+https://github.com/datastax-labs/hunter.git@8ff166979d000780ad548e49f006ef2a15d54123
certifi==2023.11.17
click==8.1.7
elastic-transport==8.11.0
elasticsearch==8.11.1
elasticsearch7==7.13.0
fmatch==0.0.4
python-dateutil==2.8.2
pytz==2023.3.post1
PyYAML==6.0.1
six==1.16.0
tzdata==2023.4
urllib3==1.26.18
```

Works for me as the requirements.txt -- any specific concerns doing it this way?